### DISCUSSION TITLE
Feature Request: Scheduler that automatically launches SelfControl according to a schedule

### DISCUSSION THREAD

**Discussion by Berkana** [SelfControlApp/selfcontrol#718] [Ideas]
I would like to request a feature: 

I love how SelfControl has helped me block off distractions, but I feel that part of me resists turning it on regularly. But this resistance would be neutralized if SelfControl were on a scheduler. If it automatically came on every weekday for a certain period of time, that would really help.

Please consider adding this feature.
Thanks in advance!

**Comment by pmrigo** [comment]
This project does exactly that:
https://github.com/andreasgrill/auto-selfcontrol

Would be fantastic as a part of SelfControl, but gets the job done.

**Reply by fahrenfort** [reply]
It doesn't work on my M1 MacBook Pro macOS Monterey unfortunately. I really hope a scheduler will be incorporated into SelfControl at some stage.
